deps/Makefile

```diff
@@ -740,6 +732,9 @@ PCRE_OBJ_TARGET = $(build_shlibdir)/libpcre2-8.$(SHLIB_EXT)
 
 # Force optimization for PCRE flags (Issue #11668)
 PCRE_CFLAGS = -O3
+ifneq ($(OS),WINNT)
+PCRE_LDFLAGS = "-Wl,-rpath,'$(build_libdir)'"
```

Comment:
Yep, same with rpmlint on the fedora/opensuse side. Pedantic, but usually not baseless.